include/OpenSpaceToolkit/Physics/Environment.hpp

```diff
@@ -57,19 +57,6 @@ class Environment
         const bool& setGlobalInstance = false
     );
 
-    /// @brief              Copy constructor
-    ///
-    /// @param              [in] anEnvironment An environment
-
-    Environment(const Environment& anEnvironment);
```

Comment:
Do you intent to use the defaults version, or forbid their use?

Reply:
Intend to use the default version.